examples/browser-to-server/README.md

```diff
@@ -0,0 +1,34 @@
+# js-libp2p-webrtc Browser to Server
+
+This example leverages the [vite bundler](https://vitejs.dev/) to compile and serve the libp2p code in the browser. You can use other bundlers such as Webpack, but we will not be covering them here.
+
+## Running the Go Server
+
+To run the Go LibP2P WebRTC server:
+
+```shell
+npm run go-libp2p-server
```

Comment:
Am I to understand correctly that this command assumes you have go installed already? And at whatever version is necessary? Is it possible npm could make sure that's true for you?